docs/reference/analysis/analyzers/whitespace-analyzer.asciidoc

```diff
@@ -4,3 +4,8 @@
 An analyzer of type `whitespace` that is built using a
 <<analysis-whitespace-tokenizer,Whitespace
 Tokenizer>>.
+
+The whitespace analyzer splits ((("whitespace analyzer")))the text on whitespace. It doesn't
```

Comment:
The `(((...)))` is for building an index, which we don't do for our docs (just the definitive guide, when going to print), so can be deleted.